spaceship/lib/spaceship/test_flight/build.rb

```diff
@@ -71,12 +71,12 @@ class Build < Base
       export_compliance_missing: 'testflight.build.state.export.compliance.missing'
     }
 
-    # Find a Build by `build_id`. Returns `nil` if can't find it.
+    # Find a Build by `build_id`.
     #
     # @return (Spaceship::TestFlight::Build)
     def self.find(app_id: nil, build_id: nil)
       attrs = client.get_build(app_id: app_id, build_id: build_id)
-      self.new(attrs) if attrs
+      self.new(attrs)
```

Comment:
Will this be safe to have this return a `Build` object without any attributes? Might cause an issue if we are checking for its presence at any point right?